src/trie/trie.c

```diff
@@ -47,8 +47,8 @@ typedef struct {
   bool stop;
 
   // timeout
+  uint32_t timeoutCounter;  // counter to limit number of calls to TimedOut()
```

Comment:
same here, let's handle these in another PR

Reply:
https://github.com/RediSearch/RediSearch/pull/7253#discussion_r2508274033